keyboards/Emi20/emi.c

```diff
@@ -0,0 +1,17 @@
+#include "emi20.h"
```

Comment:
Could you make sure that the license header is at the very top of the file?

Reply:
Huh, thought I did. maybe didn't save. Will try it again today